xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -556,14 +564,14 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 		err = cudaSetDevice(ctx->device_id);
 		if(err != cudaSuccess)
 		{
-			printf("WARNING: NVIDIA GPU %d: cannot be selected.\n", ctx->device_id);
+			printf("SKIP! (cudaSetDevice failed)\n");
```

Comment:
please revert this, device id is important

Reply:
Same as above, duplicate information, it's literally on the same line with the device_id and arch regardless if error or not (and forms a sentence...)